### ISSUE TITLE
`pip install tensorstore` fails on Python 3.11

### ISSUE THREAD

**Issue by IvyZX** [google/tensorstore#70]
Platform: macOS Version 12.6.1
Environment: Python 3.11.0
Command: `pip install tensorstore`

Output:
```
Collecting tensorstore
  Using cached tensorstore-0.1.28.tar.gz (5.4 MB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... done
Collecting numpy>=1.16.0
  Using cached numpy-1.23.5-cp311-cp311-macosx_10_9_x86_64.whl (18.1 MB)
Building wheels for collected packages: tensorstore
  Building wheel for tensorstore (pyproject.toml) ... |
```

The command is stuck at the last line `Building wheel for tensorstore`. Looks like the wheel is broken?

This is affecting dependent packages like Flax (https://github.com/google/flax/pull/2692).

**Comment by wookayin** [comment]
I submitted a PR #71 that would fix this.

**Comment by wookayin** [comment]
FYI there is no prebuilt wheel released on pypi; once #71 is released, please include py311 in the CI/CD process so the next release would have py311 wheels included. Thanks!

**Comment by laramiel** [comment]
That looks familiar; I had a fix for a similar problem a while back.

**Comment by wookayin** [comment]
Could you please make a stable release soon? Thanks.

**Comment by laramiel** [comment]
I just tagged [v0.1.29](https://github.com/google/tensorstore/tree/v0.1.29), so it should be available as soon as CI completes.

**Comment by laramiel** [comment]
... that failed; I need to increase our minimum supported python to 3.8.

**Comment by laramiel** [comment]
Ok, I've released [v0.1.30](https://github.com/google/tensorstore/tree/v0.1.30), try that out.

**Comment by IvyZX** [comment]
It worked, thank you!
